From 53eec531d051db7fff17fca036eb1345c7176348 Mon Sep 17 00:00:00 2001
From: Icenowy Zheng <icenowy@aosc.io>
Date: Wed, 1 Aug 2018 11:42:46 +0800
Subject: [PATCH 081/120] dt-bindings: simplefb-sunxi: add pipeline strings for
 DE2/3 w/ TCON_TOP

Newer Allwinner SoCs come with a display pipeline muxing component
called TCON_TOP, and the TCONs are named "tcon_lcdX" or "tcon_tvX".

Add some corresponding pipeline strings for them.

Signed-off-by: Icenowy Zheng <icenowy@aosc.io>
---
 .../devicetree/bindings/display/simple-framebuffer.yaml         | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/display/simple-framebuffer.yaml b/Documentation/devicetree/bindings/display/simple-framebuffer.yaml
index 678776b6012a..a95189232b32 100644
--- a/Documentation/devicetree/bindings/display/simple-framebuffer.yaml
+++ b/Documentation/devicetree/bindings/display/simple-framebuffer.yaml
@@ -107,6 +107,8 @@ properties:
       - mixer0-lcd0-hdmi
       - mixer1-lcd1-hdmi
       - mixer1-lcd1-tve
+      - mixer0-tcon_tv0-hdmi
+      - mixer0-tcon_lcd0"
 
   amlogic,pipeline:
     description: Pipeline used by the framebuffer on Amlogic SoCs
-- 
2.21.0

